src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -129,6 +129,12 @@ public String updateAttachment(@GraphQLRootContext Map<String, Object> context,
     }
     assertAllowedMimeType(attachment.getMimeType());
 
+    final Map<String, String> allowedClassification = (Map<String, String>) AnetObjectEngine
+        .getConfiguration().getDictionaryEntry("fields.attachment.classification.choices");
+    if (!allowedClassification.values().contains(attachment.getClassification())) {
+      throw new WebApplicationException("Classification is not allowed", Status.NOT_ACCEPTABLE);
```

Comment:
This status code has a different purpose https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

I think https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400 should be used